crates/holochain_state/src/mutations.rs

```diff
@@ -742,25 +742,24 @@ pub fn insert_entry(
     Ok(())
 }
 
-/// Lock the chain with the given lock id until the given end time.
-/// During this time only the lock id will be unlocked according to `is_chain_locked`.
-/// The chain can be unlocked for all lock ids at any time by calling `unlock_chain`.
-/// In theory there can be multiple locks active at once.
-/// If there are multiple locks active at once effectively all locks are locked
-/// because the chain is locked if there are ANY locks that don't match the
-/// current id being queried.
-/// In practise this is useless so don't do that. One lock at a time please.
+/// Lock the author's chain until the given end time.
+///
+/// The lock must have a `subject` which may have some meaning to the creator of the lock.
+/// For example, it may be the hash for a countersigning session. Multiple subjects cannot be
+/// used to create multiple locks though. The chain is either locked or unlocked for the author.
+/// The `subject` just allows information to be stored with the lock.
```

Comment:
It seems the subject is also used to identify for which preflight entry the chain is locked. Maybe mention that here?